src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.searchInput {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px #3d4e611a;
+  font-family: "Avenir", sans-serif;
+}
+
+.firstSearch {
+  display: block;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.secondSearch {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  outline: none;
+}
+
+.firstSearch:hover,
+.secondSearch:hover {
+  box-shadow: 0 3px 8px #3d4e611a;
+  outline: none;
+}
```

Comment:
Instead of applying it to both modifier classes - apply it to main class (searchInput)